course/frontend/arrays/scripts/main.js

```diff
@@ -1,96 +1,141 @@
-function isArray(arr) {
-    return arr instanceof Array;
-}
+// function isArray(arr) {
```

Comment:
UNCOMMENT previous code. all other code already pushed before should be ready to run and test as before. it should not be commented. just uncomment the code, make sure it still works fine (may you copy and paste the code commented as it is already in upstream/develop to restore the initial state of it, as it should not be manipulated). and after this, we are done a prepared for review an merge ,)